src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -372,7 +373,10 @@
                         :component status-tags/preview-status-tags}
                        {:name      :token-tag
                         :options   {:topBar {:visible true}}
-                        :component token-tag/preview-token-tag}]
+                        :component token-tag/preview-token-tag}
+                       {:name      :network-tags
```

Comment:
let's keep this alpha sorted - it's a huge file and this helps make it a bit easier to maintain 👍

Reply:
Done!